deps/v8/build/toolchain.gypi

```diff
@@ -1104,7 +1104,7 @@
           'VCCLCompilerTool': {
             'Optimization': '0',
             'conditions': [
-              ['component=="shared_library"', {
+              ['component=="shared_library" or node_shared=="true"', {
```

Comment:
I would say yes, it's a 2 line change for V8 to `toolchain.gypi`. Let me know when that's ready and I will remove the warning from the title of this PR and push it once enough LGTMs are gathered.